src/EventTickets/Migrations/CreateEventTicketsTable.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Give\EventTickets\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class CreateEventTicketsTable extends Migration {
+    /**
+     * @inheritdoc
+     */
+    public static function id() {
+        return 'give-events-create-events-tickets-table';
+    }
+
+    public static function title() {
+        return 'Create give_events table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp() {
+        return strtotime( '2022-01-29 02:00:00' );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run() {
+        global $wpdb;
+
+        $table   = $wpdb->give_event_tickets;
+        $charset = DB::get_charset_collate();
+
+        // id BINARY(16) DEFAULT (UUID_TO_BIN(UUID())),
+        $sql = "CREATE TABLE $table (
+			id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+			event_id INT UNSIGNED NOT NULL,
+			ticket_type_id INT UNSIGNED NOT NULL,
+			date_created DATETIME NOT NULL,
+			PRIMARY KEY  (id)
+		) $charset";
```

Comment:
I think we need to record which donation generated this ticket in order to retrieve the donor's information.

Reply:
I'll add a `donation_id` property and column.

Reply:
Resolved, see [381fbaa](https://github.com/impress-org/givewp/pull/7213/commits/381fbaa73e94fbd9e03d2f165d742caa477ad560).